src/_reducers/assets.reducer.js

```diff
@@ -26,16 +29,28 @@ export default (state = initialState.assets, action) => {
         assetsList: [action.payload, ...state.assetsList],
         assetsCount: state.assetsCount + 1,
         hasError: false,
-        isLoading: false
+        isLoading: false,
+        success: 'Hoooray! Asset successfully created. You can create another one or head on to view all assets.',
+        errorMessage: ''
       };
 
     case CREATE_ASSET_FAIL:
       return {
         ...state,
         hasError: true,
-        isLoading: false
+        isLoading: false,
+        success: '',
+        errorMessage: 'Oops, Asset creation unsuccessful. Please ensure the asset tag and serial number are unique then try again.'
```

Comment:
Out of curiosity, Doesn't the API return the error message?

Reply:
Actually hold on, it might be there.

Reply:
@kimotho-njoki 🤣 ...I saw your question in Slack. If they haven't fixed it yet then add a `TODO` comment, similar to what I did for the refactor and address it later. 
Yes, you should be showing the user the error messages from the API. For now add a comment as mentioned.
As for success message, you can just hardcode it